utils/buildactions/install_discord.lua

```diff
@@ -20,19 +20,16 @@ newaction {
 		if os.host() ~= "windows" then return end
 
 		-- Check md5
-		local download_discord = true
 		local archive_path = DISCORD_VENDOR.."/"..DISCORD_FILENAME
 		if os.isfile(archive_path) and os.md5_file(archive_path) == DISCORD_MD5 then
 			print("Discord Game SDK is up to date.")
-			download_discord = false
+			return
```

Comment:
It was like this before, but I found that if you delete the other files but _keep_ the `.zip`, it won't re-add the other files.

I think CEF has the same issue, so I don't mind if both do.